docs/release-notes.md

```diff
@@ -18,6 +18,17 @@
 
 # Release notes
 
+## Trino Gateway 5 (?? Jan 2024)
+
+[gateway-ha-5-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/5/gateway-ha-5-jar-with-dependencies.jar)
+
+* Add support for OAuth 2 audience use.  (https://github.com/trinodb/trino-gateway/pull/164)
```

Comment:
Maybe link to the OAuth docs for what audience use is?

Reply:
We dont have that in the Trino Gateway docs and its too specific in other docs.